packages/react-form-renderer/src/condition/condition.js

```diff
@@ -33,76 +33,73 @@ export const reducer = (state, { type, sets }) => {
   }
 };
 
-const Condition = React.memo(
-  ({ condition, children, values, field }) => {
-    const formOptions = useFormApi();
-    const dirty = formOptions.getState().dirty;
+const Condition = ({ condition, children, field }) => {
+  const formOptions = useFormApi();
+  const dirty = formOptions.getState().dirty;
 
-    const [state, dispatch] = useReducer(reducer, {
-      sets: [],
-      initial: true,
-    });
+  const [state, dispatch] = useReducer(reducer, {
+    sets: [],
+    initial: true,
+  });
 
-    // It is required to get the context state values from in order to get the latest state.
-    // Using the trigger values can cause issues with the radio field as each input is registered separately to state and does not yield the actual field value.
-    const conditionResult = parseCondition(condition, formOptions.getState().values, field);
+  // It is required to get the context state values from in order to get the latest state.
+  // Using the trigger values can cause issues with the radio field as each input is registered separately to state and does not yield the actual field value.
+  const conditionResult = useMemo(() => parseCondition(condition, formOptions.getState().values, field), [formOptions.getState().values, condition]);
```

Comment:
`field` is used inside the code block and should it be listed as one of the dependencies?

Reply:
Yeah, I was thinking about that a lot. We use `field` there to only pass `field.name` to conditions in case of dynamic forms and `name` cannot be changed anyways.

However, I added there and then tested it - performance still looks fine, so fixed. Thank you for pointing this out! 😊